hugegraph-rocksdb/src/main/java/org/apache/hugegraph/backend/store/rocksdb/RocksDBTable.java

```diff
@@ -359,6 +353,7 @@ public long estimateNumKeys(RocksDBSessions.Session session) {
 
         @Override
         public byte[] position(String position) {
+            // TODO: START & END is same & be empty now? remove one?
```

Comment:
the values are equaled, but the meanings are different, we can add NOTE for the const define